aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -177,7 +221,12 @@ impl Aggregator {
             },
             AggregatorState::PositiveDelta => {
                 // If positive delta, add directly but also record the state.
-                self.value = addition(self.value, value, self.limit)?;
+                self.value = addition(self.value, value, self.limit).map_err(|err| {
+                    if self.value < u128::MAX - value {
+                        self.record_overflow(self.value + value);
```

Comment:
I think we should record limit here?

Reply:
What do you mean by recoding limit? `self.value + value` is the value of delta for which the overflow has occurred right?